Formula/wine.rb

```diff
@@ -8,16 +8,9 @@ class Wine < Formula
   head "git://source.winehq.org/git/wine.git"
 
   stable do
-    url "https://dl.winehq.org/wine/source/1.8/wine-1.8.6.tar.bz2"
-    mirror "https://downloads.sourceforge.net/project/wine/Source/wine-1.8.6.tar.bz2"
-    sha256 "b1797896eb3b63aab8a4753cc756d6211a0e85460146a1b52063ec79c13906d3"
-
-    # Patch to fix screen-flickering issues. Still relevant on 1.8. Broken on 1.9.10.
-    # https://bugs.winehq.org/show_bug.cgi?id=34166
-    patch do
-      url "https://bugs.winehq.org/attachment.cgi?id=52485"
-      sha256 "59f1831a1b49c1b7a4c6e6af7e3f89f0bc60bec0bead645a615b251d37d232ac"
```

Comment:
Should this patch actually be removed, or just replaced with https://bugs.winehq.org/attachment.cgi?id=55968 ?